tests/projects/utils.py

```diff
@@ -39,9 +40,16 @@ def assert_dirs_equal(expected, actual):
 def build_docker_example_base_image():
     print(os.path.join(TEST_DOCKER_PROJECT_DIR, 'Dockerfile'))
     client = docker.from_env()
-    client.images.build(tag='mlflow-docker-example', forcerm=True,
-                        dockerfile='Dockerfile',
-                        path=TEST_DOCKER_PROJECT_DIR)
+    try:
+        client.images.build(tag='mlflow-docker-example', forcerm=True,
+                            dockerfile='Dockerfile', path=TEST_DOCKER_PROJECT_DIR)
+    except BuildError as build_error:
+        for chunk in build_error.build_log:
+            print(chunk)
+        raise build_error
+    except APIError as api_error:
+        print(api_error.explanation)
+        raise api_error
```

Comment:
could there be other errors?

Reply:
i says the following are raised. `TypeError` would be just retrown

        Raises:
            :py:class:`docker.errors.BuildError`
                If there is an error during the build.
            :py:class:`docker.errors.APIError`
                If the server returns any other error.
            ``TypeError``
                If neither ``path`` nor ``fileobj`` is specified.